include/trusted-firmware-a.mk

```diff
@@ -1,12 +1,21 @@
 PKG_NAME ?= trusted-firmware-a
 PKG_CPE_ID ?= cpe:/a:arm:trusted_firmware-a
 
+PKG_LTS ?=
+
+ifneq ($(PKG_LTS),)
+PKG_VERSION_PREFIX:=lts-v
+PKG_BUILD_DIR = $(BUILD_DIR)/$(PKG_NAME)-$(BUILD_VARIANT)/$(PKG_NAME)-$(PKG_VERSION_PREFIX)$(PKG_VERSION)
+else
+PKG_VERSION_PREFIX:=v
+PKG_BUILD_DIR = $(BUILD_DIR)/$(PKG_NAME)-$(BUILD_VARIANT)/$(PKG_NAME)-$(PKG_VERSION)
+endif
+
 ifndef PKG_SOURCE_PROTO
-PKG_SOURCE = trusted-firmware-a-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://codeload.github.com/TrustedFirmware-A/trusted-firmware-a/tar.gz/v$(PKG_VERSION)?
+PKG_SOURCE = trusted-firmware-a-$(_PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/TrustedFirmware-A/trusted-firmware-a/tar.gz/$(PKG_VERSION_PREFIX)$(PKG_VERSION)?
 endif
 
-PKG_BUILD_DIR = $(BUILD_DIR)/$(PKG_NAME)-$(BUILD_VARIANT)/$(PKG_NAME)-$(PKG_VERSION)
```

Comment:
Why not always using:
```
PKG_BUILD_DIR = $(BUILD_DIR)/$(PKG_NAME)-$(BUILD_VARIANT)/$(PKG_NAME)-$(PKG_VERSION_PREFIX)$(PKG_VERSION)
```

Reply:
This is because the tags are either e.g. lts-v2.12.1 or v2.12. But when downloading the file from Github in a browser, the filenames are:
```
trusted-firmware-a-2.12.tar.gz
trusted-firmware-a-lts-v2.12.1.tar.gz
```
This is reflected in the directory in the tarball:
```
➜ tar --list --file dl/trusted-firmware-a-v2.12.tar.gz  | head -n1
trusted-firmware-a-2.12/
➜ tar --list --file dl/trusted-firmware-a-lts-v2.12.1.tar.gz  | head -n1
trusted-firmware-a-lts-v2.12.1/
```

So we need to specify the v prefix when downloading from codeload, but we need to exclude it in PKG_BUILD_DIR.

Reply:
I'll add this to the commit message.